app/routes.py

```diff
@@ -1,2 +1,221 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, make_response, request
+from sqlalchemy import asc, desc
+from datetime import datetime, timezone
+import requests
+import os
 
+SLACK_POST_MESSAGE_URL = 'https://slack.com/api/chat.postMessage'
+SLACK_POST_MESSAGE_CHANNEL = '#task-list'
+SLACK_BOT_TOKEN = os.environ.get('SLACK_TOKEN')
+SLACK_BOT_USERNAME = 'Waebot'
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+def make_input_valid(number):
+    try:
+        int(number)
+    except:
+        return make_response(f"{number} is not an int!", 400)
```

Comment:
This is cleaner if you just use `abort`:

```suggestion
        abort(make_response(f"{number} is not an int!", 400))
```

(`abort` raises a special exception that results in a response being sent back immediately.)